warehouse/query-core/src/main/java/datawave/query/jexl/visitors/RangeConjunctionRebuildingVisitor.java

```diff
@@ -112,7 +112,7 @@ public RangeConjunctionRebuildingVisitor(ShardQueryConfiguration config, Scanner
     
     @Override
     public Object visit(ASTReference node, Object data) {
-        if (ASTDelayedPredicate.instanceOf(node) || IndexHoleMarkerJexlNode.instanceOf(node) || ASTEvaluationOnly.instanceOf(node)) {
+        if (IndexHoleMarkerJexlNode.instanceOf(node) || ASTEvaluationOnly.instanceOf(node)) {
```

Comment:
Similar to comment above.  We could probably make this run faster by doing one run of the visitor and setting a list with IndexHoleMarker and EvaluationOnly as the two types we check for.  Not a big deal though.

Reply:
agree